test/e2e-common.sh

```diff
@@ -20,9 +20,9 @@ source $(dirname $0)/../vendor/knative.dev/test-infra/scripts/e2e-tests.sh
 # Latest serving release. This is intentionally hardcoded for now, but
 # will need the ability to test against the latest successful serving
 # CI runs in the future.
-readonly LATEST_SERVING_RELEASE_VERSION=0.6.0
+readonly LATEST_SERVING_RELEASE_VERSION=$(git describe --match "v[0-9]*" --abbrev=0)
 # Istio version we test with
-readonly ISTIO_VERSION=1.1.3
+readonly ISTIO_VERSION="1.4.2"
```

Comment:
is there a way to determine this dynamically? the version should be in a file in third_party/?

Reply:
1.4-latest is taken as default, if this param is not set.
HOWEVER, the directory is not available under the path, https://raw.githubusercontent.com/knative/serving, but it is available under the source code folder.
It only happens to this 1.X-latest directory, not to others like 1.4.2.